readme.md

```diff
@@ -10,7 +10,7 @@ See the [readme.txt](readme.txt) for installation and usage instructions.
 
 ## Contribute
 
-Please [report issues](https://github.com/WordPress/two-factor/issues) and [open pull requests](https://github.com/WordPress/two-factor/pulls) on GitHub.
+Please [report (non-security) issues](https://github.com/WordPress/two-factor/issues) and [open pull requests](https://github.com/WordPress/two-factor/pulls) on GitHub. See below for information on reporting potential security/privacy vulnerabilities.
```

Comment:
Do you want to mimic this section in `readme.txt` so it appears on the dotorg plugin page?

Reply:
That sounds good 👍🏻 [How does it look now](https://github.com/WordPress/two-factor/pull/481/files)?

Reply:
I'm gonna go ahead and mark this as resolved and merge, but we can definitely iterate if you have any feedback